proto/astria/sequencer/v1alpha1/transaction.proto

```diff
@@ -75,3 +76,18 @@ message MintAction {
   bytes to = 1;
   astria.primitive.v1.Uint128 amount = 2;
 }
+
+message Ics20Withdrawal {
```

Comment:
Can you add comments to this type, what IBC spec it matches, maybe with links. and how this is used?

Please also cover the fields and their constraints, i.e. the format of the strings, the number of bytes for things like the return address, units of the time.

Reply:
yep, added more description, and added a longer comment + spec link to the rust type